docs/static/transforming-data.asciidoc

```diff
@@ -450,7 +450,79 @@ For example:
     }
 --------------------------------------------------------------------------------
 
-<<plugins-filters-jdbc_streaming,jdbc_streaming>>::
+
+<<plugins-filters-jdbc_static,jdbc_static filter>>::
+
+Enriches events with data pre-loaded from a remote database.
++
+The following example fetches data from a remote database, caches it in a local
+database, and uses lookups to enhance events with data cached in the local
+database.
++
+[source,json]
+-----
+filter {
+  jdbc_static {
+    loaders => [
+      {
+        id => "remote-servers"
+        query => "select ip, descr from ref.local_ips order by ip"
+        local_table => "servers"
+      },
+      {
+        id => "remote-users"
+        query => "select firstname, lastname, userid from ref.local_users order by userid"
+        local_table => "users"
+      }
+    ]
+    local_db_objects => [
+      {
+        name => "servers"
+        index_columns => ["ip"]
+        columns => [
+          ["ip", "varchar(15)"],
+          ["descr", "varchar(255)"]
+        ]
+      },
+      {
+        name => "users"
+        index_columns => ["userid"]
+        columns => [
+          ["firstname", "varchar(255)"],
+          ["lastname", "varchar(255)"],
+          ["userid", "int"]
+        ]
+      }
+    ]
+    local_lookups => [
+      {
+        id => "local-servers"
+        query => "select descr as description from servers WHERE ip = :ip"
+        parameters => {ip => "[from_ip]"}
+        target => "server"
+      },
+      {
+        id => "local-users"
+        query => "select firstname, lastname from users WHERE userid = :id"
+        parameters => {id => "[loggedin_userid]"}
+        target => "user"
+      }
+    ]
+    # using add_field here to add & rename values to the event root
+    add_field => { server_name => "%{[server][0][description]}" }
+    add_field => { user_firstname => "%{[user][0][firstname]}" }
+    add_field => { user_lastname => "%{[user][0][lastname]}" }
+    remove_field => ["server", "user"]
+    jdbc_user => "logstash"
```

Comment:
@dedemorton @guyboertje we should also document how to use the loader_schedule option to refresh the cache. not specifying it would mean only one initial load.

https://github.com/logstash-plugins/logstash-filter-jdbc_static/blob/master/lib/logstash/filters/jdbc_static.rb#L80

Reply:
That's documented in the reference topic. We should avoid putting too much detail in this topic because it's meant to be an intro, not the main documentation. This topic points off to the [main topic](https://www.elastic.co/guide/en/logstash/6.2/plugins-filters-jdbc_static.html#plugins-filters-jdbc_static-loader_schedule) for the nitty gritty details.